### PR TITLE
v1.0.0

### PR DESCRIPTION
**Pull request by d-cogswell**
We've implemented a lot of major changes and it's time for a major release update. What else should be addressed first?
- Add documentation for the new CCCVCPcycle profileType
- Documentation updates for daetools installation
- Should we switch back to the official daetools for testing since it has been updated?

### CODE REVIEW

**Comment by v1kko** [comment]
>     * Should we switch back to the official daetools for testing since it has been updated?

I agree with this. I can have a look tomorrow / next week, hopefully the changes to the workflow are simple and minimal :)

**Comment by v1kko** [comment]
I have added a regression tests with the newer daetools from sourceforge, I left the old ones in place as the one from sourceforge only seems to work with python 3.10 . ( it doesn't work out of the box and I don't have time to fix it).

**Review by v1kko — APPROVED** [review]
I added a regression testing which uses the sourceforge version of daetools, that part I now approve :)

**Comment by d-cogswell** [comment]
Development branch documentation is now automatically built here: https://mpet.readthedocs.io/en/development/

**Comment by d-cogswell** [comment]
@loostrum does it make sense to have dask and dash requirements as `extras_require` in setup.py?

**Comment by laurasootes** [comment]
> @loostrum does it make sense to have dask and dash requirements as `extras_require` in setup.py?

@d-cogswell, you are right, it makes more sense to move the dash and dask requirements to extras_require

**Comment by d-cogswell** [comment]
Thanks @laurasootes and @loostrum

File: tests/README.md [hunk 1]
```diff
@@ -62,3 +62,5 @@ it is both tested as absolute and relative tolerance.
  - test021: hdf5Fast file output and restarting hdf5 simulations
  - test022: Test of specified_psd_c option, LFP homog
  - test023: CIET for LFP
+ - test028: CCCVCPcycling for full cell with LIONSIMBA test file
+ - test029: CCCVCPcycling for half cell with LIONSIMBA test file
```
Comment (lightningclaw001) [inline tests/README.md:66]: are we adding any test cases for the solid electrolyte? and for the parallel jobs

Comment (loostrum) [reply tests/README.md:66]: test025 and test026 are for solid electrolyte (at least those are two we added). Let me know if you want me to add that to the tests readme file.

**Pull request merged**